### PR TITLE
feat(repeat): add repeat feature

### PR DESCRIPTION
**Pull request by lazav94**

### CODE REVIEW

**Comment by lazav94** [comment]
Should be done

File: packages/repeat/src/index.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/**
+ * Repeats the input text number of times.
+ * @param {String} [text] The input text.
+ * @returns {Integer} times How many times shall we repeat the text, default to 1
+ * @param {String} glue The glue for the output, default to ''
```
Comment (vorillaz) [inline packages/repeat/src/index.js]: I love the naming here 🗡

File: packages/repeat/src/index.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+ * @returns {String} The repeated string;
+ */
+const repeat = (text, repeat = 1, glue = '') => {
```
Comment (vorillaz) [inline packages/repeat/src/index.js]: The function and the variable are both name `repeat`. 😬

Comment (lazav94) [reply packages/repeat/src/index.js]: I changed param name form 'repeat' to 'times'.

File: packages/repeat/src/index.js (hunk 1 above)
Comment (lazav94) [reply packages/repeat/src/index.js]: I like your solution, it's more elegant but is a little slower.  
I run a local performance test and when I try 100k repeat you can see the difference of execution.

Comment (vorillaz) [reply packages/repeat/src/index.js]: That's awesome @lazav94, I put mine as a point of reference and just exposing an alternative :)

**Comment by vorillaz** [comment]
@lazav94 Thanks so much for the contribution, merging.

**Pull request merged**